target_chains/ethereum/contracts/forge-test/Pyth.WormholeMerkleAccumulator.t.sol

```diff
@@ -229,12 +303,22 @@ contract PythWormholeMerkleAccumulatorTest is
         pyth.updatePriceFeeds{value: updateFee}(updateData);
         assertPriceFeedMessageStored(priceFeedMessages1[0]);
 
+        bytes[] memory updateData1 = updateData;
+
         (updateData, updateFee) = createWormholeMerkleUpdateData(
             priceFeedMessages2
         );
         pyth.updatePriceFeeds{value: updateFee}(updateData);
         // Make sure that the old value is still stored
         assertPriceFeedMessageStored(priceFeedMessages1[0]);
+
+        bytes32[] memory priceIds = new bytes32[](1);
+        priceIds[0] = priceFeedMessages1[0].priceId;
+
+        PythStructs.PriceFeed[] memory priceFeeds = pyth.parsePriceFeedUpdates{
+            value: updateFee
+        }(updateData1, priceIds, 0, MAX_UINT64);
+        assertParsedPriceFeedStored(priceFeeds[0]);
```

Comment:
this test doesn't seem like it's doing anything useful -- it's testing that this function parses a single update, which is covered by many different tests already. i would delete the change here.